highfive/configs/rust-lang/rust.json

```diff
@@ -36,7 +36,8 @@
         "rustdoc": [
             "@jsha",
             "@GuillaumeGomez",
-            "@CraftSpider"
+            "@CraftSpider",
+            "@notriddle",
```

Comment:
Unfortunately, I seem to recall JSON doesn't support trailing commas, unless highfive's JSON parser has an extension... :/

Reply:
Oh right, I forgot about that... Thanks!